test/relations.test.js

```diff
@@ -159,12 +162,20 @@ describe('relations', function() {
             var chapters = book.chapters();
             chapters.should.eql(ch);
 
-            book.chapters({order: 'name DESC'}, function(e, c) {
+            var nameFilter = isCassandraConnector ? {} : {order: 'name DESC'};
```

Comment:
AFAICT, the test case covered by this test should not require the `order` filter. Let's rewrite the test so that the same code can be used for all connectors then.

```js
book.chapters(function(e, c) {
  if (e) return done(e);
  const names = c.map(c => c.name);
  names.should.have.lengthOf(3);
  names.should.containEql(['a', 'c', 'z']);
  done();
});
```